analyzer.go

```diff
@@ -104,18 +104,23 @@ func (a *Analyzer) NotifyReviewEvent(ctx context.Context, e *pb.ReviewEvent) (
 			continue
 		}
 
-		// analyze only changes in Golang
-		if strings.HasPrefix(strings.ToLower(change.Head.Language), "go") {
-			tryToSaveTo(change.Head, tmp)
+		file := change.Head
+		if err = saveTo(file, tmp); err != nil {
```

Comment:
not important here but usually people write it like this:
```go
if err := fn(); err != {
...
```
so the `err` is defined/redefined only inside `if-else` block that handles this error,